src/group-configurations/content-groups-section/ContentGroupContainer.jsx

```diff
@@ -0,0 +1,151 @@
+import PropTypes from 'prop-types';
+import { Formik } from 'formik';
+import * as Yup from 'yup';
+import { useIntl } from '@edx/frontend-platform/i18n';
+import {
+  Alert,
+  ActionRow,
+  Button,
+  Form,
+  OverlayTrigger,
+  Tooltip,
+} from '@edx/paragon';
+
+import { WarningFilled as WarningFilledIcon } from '@edx/paragon/icons';
+
+import PromptIfDirty from '../../generic/PromptIfDirty';
+import { isAlreadyExistsGroup } from './utils';
+import messages from './messages';
+
+const ContentGroupContainer = ({
+  isEditMode,
+  groupNames,
+  isUsedInLocation,
+  overrideValue,
+  onCreateClick,
+  onCancelClick,
+  onDeleteClick,
+  onEditClick,
+}) => {
+  const { formatMessage } = useIntl();
+  const initialValues = { newGroupName: overrideValue };
+  const validationSchema = Yup.object().shape({
```

Comment:
[nit]: I propose to move this validation scheme into a separate `validation.js` file